my-app/src/About.css

```diff
@@ -0,0 +1,99 @@
+@import url(http://fonts.googleapis.com/css?family=Lato:400,700);
+
+body
+{
+  font-family: 'Lato', 'sans-serif';
+}
+
+#sect
+{
+  border-bottom: 2px solid;
+  display: relative;
+  padding-bottom: 20px;
+}
+
+.about-tile {
+  text-align: center;
+  margin-bottom: 36px;
+}
+
+.about-tile img {
+  margin: auto;
+  width: 90%;
+  display: block;
+}
+
+/* Hiding span texts for the image tiles.
+   There is already text on the images, so the text here is redundant. */
+.about-tile span {
+  display: none;
+}
+
+#about-jumbotron {
+  background-color: #fffed1;
+}
+
+#about-jumbotron a {
+  color: #2c7cff;
+}
+
+
+
+/* Is all of this needed after here? I don't see where it is used. */
```

Comment:
Adrian, are the css properties listed after this line all needed? I couldn't really see where they were being used for the most part. If not, might as well delete them to keep it as clean as possible.

Reply:
They aren't actually, we can delete them if we have time.